frontend/src/styles/global.scss

```diff
@@ -5,6 +5,8 @@ $color-pack: false;
 @import "./font-awesome/scss/solid.scss";
 @import "./font-awesome/scss/brands.scss";
 
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code&display=swap');
```

Comment:
Can you store this font in assets? So we don't have a dependency on an internet connection